style.css

```diff
@@ -496,3 +501,605 @@ button:active {
 .font-poppins {
   font-family: Poppins, sans-serif;
 }
+
+/* DESKTOP-VERSION */
+
+@media only screen and (min-width: 992px) {
+  body {
+    font-family: poppins, sans-serif, Arial, Helvetica;
+    background-color: #e5e5e5;
+    margin: 0;
+    padding: 0;
+    display: grid;
+    width: 100%;
+  }
+
+  .cover {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    height: 916px;
+    width: 100%;
+    padding-left: 0;
+    margin-right: 0;
+    background-color: #fff;
+    border-radius: 1px 1px 1px 50px;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 0;
+    margin-left: 0;
+    height: 72px;
+    width: 100%;
+    gap: 32px;
+    align-items: center;
+  }
+
+  .contact-message a {
+    height: 24px;
+    width: 100%;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #7f8cff;
+    text-decoration: none;
+    margin-left: 260px;
+    margin-top: 12px;
+  }
+
+  .nav-bar-links {
+    display: flex;
+    box-sizing: border-box;
+    justify-content: space-evenly;
+    height: 40px;
+    width: 100%;
+    margin-right: 142px;
+    padding: 10px;
+    align-items: center;
+    justify-items: end;
+  }
+
+  .nav-bar-links li {
+    text-decoration: none;
+    list-style: none;
+  }
+
+  .logo a {
+    text-decoration: none;
+  }
+
+  .social-media a {
+    /* list-style: none; */
+    color: #505f79;
+  }
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly fix it. :writing_hand: